Documentation/gettingstarted.rst

```diff
@@ -983,40 +987,34 @@ You should see output similar to the following:
 
     $ curl -i -H 'Content-Type: application/json' -d @web-server.json 127.0.0.1:8080/v2/apps
     HTTP/1.1 201 Created
-    [...]
+    ...
     Marathon-Deployment-Id: [UUID]
-    [...]
-
-Confirm that Cilium sees the new workload. The output should return the endpoint with label ``mesos:id=web-server`` and the assigned IP:
+    ...
 
+Confirm that Cilium sees the new workload: 
 ::    
 
     $ cilium endpoint list 
 
-And observe similar output to the following:
+The output should return the endpoint with label ``mesos:id=web-server`` and the assigned IP:
 
 ::
 
-    vagrant@vagrant:~$ cilium endpoint list
+    $ cilium endpoint list
     ENDPOINT   POLICY        IDENTITY   LABELS (source:key[=value])   IPv6                   IPv4           STATUS   
                ENFORCEMENT                                                                                           
     29898      Disabled      256        mesos:id=web-server           f00d::a00:20f:0:74ca   10.15.242.54   ready
 
-Use the IP from the previous output to test the web-server:
+Test the web-server provides OK output:
 
 ::    
 
-    $ curl <IP>:8181/api 
-
-The output should simply return:
-
-::
-
-    $ curl <IP>:8181/api 
+    $ export WEB_IP=`cilium endpoint list | grep web-server | awk '{print $6}'`
+    $ curl $WEB_IP:8181/api
     OK
 
 
-Run a script to create "good client" and "bad client" requests to the newly created web-server. The script should generate ``goodclient.json`` and ``badclient.json`` files for the client tasks, respectively:
+Run a script to create "good client" and "bad client" requests to the newly created web-server. The script will generate ``goodclient.json`` and ``badclient.json`` files for the client tasks, respectively:
```

Comment:
really minor, but this almost makes it sounds like the user should know what "goodclient" and "badclient" means.   Similar with the statement below that "Then submit the client tasks to Marathon, which will generate GET /public and GET /private requests:" .    I don't think people know what /public + /private are referring yet.  I might just say:      

Run a script to create two Marathan tasks ("good client" and "bad client") that will access the webserver.  We will later use the output of these test clients to validate that Cilium network policy is being correctly enforced.

Reply:
note: I'm definitely fine shipping this without this change.

Reply:
Updated.